include/spirv/unified1/spirv.core.grammar.json

```
@@ -5967,6 +5967,93 @@
       "extensions" : [ "SPV_NV_shader_image_footprint" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpTypeCooperativeVectorNV",
+      "class"  : "Reserved",
+      "opcode" : 5288,
+      "operands" : [
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef",        "name" : "'Component Type'" },
+        { "kind" : "IdRef",        "name" : "'Components'" }
```

nit w.r.t. extension's name

```suggestion
        { "kind" : "IdRef",        "name" : "'Component Count'" }
```